extension/scripts/features/hidecasinogames/ttHideCasinoGames.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+(async () => {
+	featureManager.registerFeature(
+		"Hide Casino Games",
+		"casino",
+		() => !!settings.hideCasinoGames.length,
+		null,
+		hideCasinoGames,
+		unhideCasinoGames,
+		{
+			storage: ["settings.pages.sidebar.collapseAreas"],
+		},
+		async () => {
+			if (await checkMobile()) return "Not supported on mobile!";
+
+			await requireSidebar();
+
+			if (document.find("#sidebarroot .tablet")) return "Already collapsible.";
```

Comment:
You copied a bit too much xD

Reply:
lol removed it.
It is supported on mobile. Tested it in chrome emulation.